docs/docs/user/references/defaults.md

```diff
@@ -0,0 +1,108 @@
+# Defaults
+
+KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults){target=_blank}. This file provides the user with the power to set defaults for any and all [components](/user/references/components){target=_blank}, thus omitting the need to repeat the same settings in [`pipeline.yaml`](/resources/pipeline-components/pipeline){target=_blank}.
+
+[See real-world examples for `defaults`](/resources/examples/defaults){target=_blank}.
+
+??? "KPOps component Hierarchy"
+    --8<--
+    ./docs/resources/architecture/components-hierarchy.md
+    --8<--
```

Comment:
Do we want the figure in the info box?

Reply:
I prefer it like that. "Defaults "is not the main page that discusses components, so their relationships are better explained in "Components" where the pic is not collapsed. 

imo it's nice to have the figure in "Defaults" for a quick look, but it is redundant to have it full-sized by default.